demisto_sdk/commands/split/ymlsplitter.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
what is this?

Reply:
Allows using `|` for typing instead of`Union` in Python versions 3.7-3.9 (without it, it's only possible on versions 3.10+)